src/styles/blocks/catalog.scss

```diff
@@ -2,24 +2,32 @@
   display: flex;
   flex-wrap: wrap;
 
-  max-width: 944px;
+  max-width: $catalog-width;
   margin: 0 auto;
-  padding: 50px 0;
+  padding: $catalog-padding-y 0;
 
   &__card {
     display: flex;
     justify-content: center;
     align-items: center;
     box-sizing: border-box;
 
-    width: 200px;
+    width: $card-width;
     height: 300px;
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
 
+    transition-duration: $trans-dur;
+    transition-timing-function: $smooth;
```

Comment:
in general, it is good to use the properties individually, but you can also write in shortened form

Reply:
Sorry, I did not get what you are showing me by turning off the margins and how do fonts relate to this...
Can you explain more in detail what you mean?

as for the short form, it is not as readable as individual properties.

Reply:
your transition is also applied to your margins, but it should affect only your card size and font colors

such transitions may appear at unnecessary moments (as in the example with margins) and degrade performance due to permanent calculations of these properties

Reply:
try to specify which properties should use transitions with `transition-property`

Reply:
actually @ruslanpashkov is right, you should specify exact properties you want to transition